tests/uuid_tests.rs

```diff
@@ -39,18 +39,33 @@ pub async fn insert_metadata(db: &DatabaseConnection) -> Result<(), DbErr> {
         .one(db)
         .await?;
 
-    assert_eq!(
-        json,
-        Some(json!({
-            "uuid": metadata.uuid,
-            "type": metadata.ty,
-            "key": metadata.key,
-            "value": metadata.value,
-            "bytes": metadata.bytes,
-            "date": metadata.date,
-            "time": metadata.time,
-        }))
-    );
+    if cfg!(feature = "sqlx-mysql") {
+        assert_eq!(
+            json,
+            Some(json!({
+                "uuid": metadata.uuid,
+                "type": metadata.ty,
+                "key": metadata.key,
+                "value": 1.18,
+                "bytes": "\u{1}\u{2}\u{3}",
```

Comment:
everything else looks good... except this. bytes (`Vec<u8>`) now becomes a string ?

Reply:
I reverted the assertions change. And modified the `into_json` logic instead, https://github.com/SeaQL/sea-orm/pull/2305/commits/6cda01e5505213d9c290a01e0320d1fdace38978.